### DISCUSSION TITLE
磁盘管理器布局，不能一次显示全部分区信息

### DISCUSSION THREAD

**Discussion by a137933** [linuxdeepin/developer-center#3204] [Features Request & Ideas | 特性请求 & 头脑风暴]
如图，极其鸡肋，具体看帖
https://club.uniontech.com/ucare/#/detail?id=05d6c26731a64352b9a0142a3ba8cb4d
[image.png…](https://storage.deepin.org/thread/202101221015336258_1.png)

**Comment by BLumia** [comment]
讲道理，深度分区管理工具出了也没多久，并且还在不断开发需求，目前没 gparted 好使也挺正常的...

我标个需求标签吧。

**Reply by a137933** [reply]
我记得是2020年10月就出了，挺久的了。
gparted，深度和uos已经不自带了。
把自己的产品搞好才是真，毕竟自带。
